modules/remove-central-attached-pv-overview.adoc

```diff
@@ -3,14 +3,14 @@
 // * upgrading/upgrade-operator.adoc
 :_mod-docs-content-type: CONCEPT
 [id="remove-central-attached-pv-overview_{context}"]
-= Remove Central-attached PV
+= Remove Central-attached PV after upgrading to version 4.1 and later
 
 [role="_abstract"]
 Kubernetes and {ocp} do not delete persistent volumes (PV) automatically. When you upgrade {product-title-short} from earlier versions, the Central PV called `stackrox-db` remains mounted. However, in {product-title-short} 4.1, Central does not need the previously attached PV anymore.
 
 The PV has data and persistent files used by earlier {product-title-short} versions. You can use the PV to roll back to an earlier version before {product-title-short} 4.1. Or, if you have a large RocksDB backup bundle for Central, you can use the PV to restore that data.
 
-If you do not plan to roll back or restore from earlier RocksDB backups, you can remove the Central-attached persistent volume claim (PVC) to free up the storage.
+After completing the upgrade to 4.1, you can remove the Central-attached persistent volume claim (PVC) to free up the storage if you do not plan to roll back or restore from earlier RocksDB backups.
```

Comment:
Thinking of something like this to make the language more active and simplify the first verb phrase: "After you upgrade RHACS to 4.1, you can remove the Central-attached persistent volume..."

Reply:
Might consider breaking up the sentence. It's quite long.